src/Preference.h

```diff
@@ -8,11 +8,18 @@
 #include "RageUtil.h"
 class XNode;
 
+enum class PreferenceFlag
+{
+	Default = 0,
+	Immutable = 1,
+	Compat = 2,

```

Comment:
Add someone documentation on what each of these enum values means

Reply:
Done!